system/lib/llvm-libc/src/__support/StringUtil/tables/posix_errors.h

```diff
@@ -63,7 +63,12 @@ LIBC_INLINE_VAR constexpr MsgTable<76> POSIX_ERRORS = {
     MsgMapping(EPROTO, "Protocol error"),
     MsgMapping(EMULTIHOP, "Multihop attempted"),
     MsgMapping(EBADMSG, "Bad message"),
+#ifdef __EMSCRIPTEN__
+    // For now, match the musl string
```

Comment:
Is this because there is some tests that cares about the exact string?

Reply:
I guess if we switch over for real, we can just revert these changes and update the tests?